CONTRIBUTING.rst

```diff
@@ -24,6 +24,15 @@ message guidelines described below:
 
 Please provide your patches as GitHub pull requests. Thanks!
 
+Rerunning failed CI workflows
+-------------------------
+
+* Ask the maintainers to add the ``ok-to-test`` label on the PR
```

Comment:
Is there a way to avoid this? Not sure if it's hardcoded in the GH Action, I haven't looked into it in depth. We already have to approve workflow runs from first-time contributors, so if that was enough it'd be great. For regular contributors we can assume we trust them.

Reply:
Good point. I believe there is a way to automate the process. Let's examine it as a follow up. WDYT?

Reply:
Yeah. I've noticed it as well.